solidity/migrations/prices.json

```diff
@@ -1,4 +1,3 @@
 {
-    "BTCUSD": "7152.55",
-    "ETHUSD": "142.28"
+    "BTCETH": "502709446162"
```

Comment:
Is this made up or correct? Keep in mind this was originally set via https://github.com/keep-network/tbtc/blob/61822b0fcea66b4f0d3b51d4d54a7c8301f90222/solidity/scripts/update-prices.sh . May make sense to update that to work for the one value want here and set it to a recent value for completeness.

Reply:
it's correct given   `"BTCUSD": "7152.55"` and` "ETHUSD": "142.28"`
updating the script makes sense

Reply:
… Is it?

```
  1 BTC     $142.28     
--------- x ------- = 0.0198 BTC / ETH = 50.270 ETH / BTC
$7152.55     1 ETH  
```

Is the issue here that BTCETH is really “ETHsats”?

Reply:
No, I got my denominations wrong, sorry. Need to just do this math for a week or two instead of coming back to it once every 3 months hah. This is “weiBTC”, yah?

Reply:
yeah. The math is correct, it's just `x 10^10`

Reply:
Wait, sorry. should be `weiSat`, not `weiBtc`. 
`x 10^10` is the result of simplifying `x 10^8`(for sat) and `x 10^18` (for wei)

Reply:
Ok. I'm double-checking to see if we'll have weiBTC, weisats, or something else.

Reply:
I too made this "units conversion" mistake in https://github.com/keep-network/tbtc/pull/393. I'd definitely like to avoid the confusion where we can. Maybe add a doc if we're changing from decimal point ETH to wei?

Reply:
I would love for this to be referred to as `weisat` if that's what we've got, rather than `ETHBTC` (or `BTCETH` which is the wrong way around).

I haven't heard back from Maker yet on the denomination they've chosen to use.

Reply:
> `BTCETH` which is the wrong way around

Why is this the wrong way around?

shouldn't `BTCETH` return `BTC` value in terms of `ETH`
same way `ETHUSD` returns `ETH` in terms of `USD`?

Reply:
Oh hey you're right. I keep reading these as “ETH per USD” rather than “ETH in USD”. I'll unbreak my brain, my bad.

Looks like the [Kovan ETHBTC Medianizer](https://kovan.etherscan.io/address/0xf60df9b138a00ae8dbd07f55fd2305cc791e68cc#events) is publishing BTC in wei. So, quick take: `weiBTC` is what we call our thing, adjust accordingly throughout.

Reply:
unfortunately, we'll have a naming split between weiBTC and BTCwei. since we're given one and need the other

Reply:
`satwei` actually

Reply:
So, we need weisat (number of wei in 1 sat), since lot size is in sats and we'll want bond amount in wei, right? And the feed will give us BTCwei. And `10**26/BTCwei` will give us `weisat`, ya?

Reply:
I'm a dumb and got this backwards again. Feed is ETHBTC, correctly---BTC in 1 ETH, 18 decimals. We want satwei---wei in 1 sat, also 18 decimals. `10**26/ETHBTC` gives `satwei`.